Controllers/HandleKeyboard.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  HandleKeyboard.swift
+//  Clic&Farm-iOS
+//
+//  Created by Jonathan DE HAAY on 14/09/2018.
+//  Copyright © 2018 Ekylibre. All rights reserved.
+//
+import UIKit
+
+extension UIViewController {
+  func hideKeyboardWhenTappedAround() {
+    let tap = UITapGestureRecognizer(target: self, action: #selector(UIViewController.dismissKeyboard))
+
+    tap.cancelsTouchesInView = false
+    view.addGestureRecognizer(tap)
+  }
+
+  func moveViewWhenKeyboardAppears() {
+    NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillShow), name: UIResponder.keyboardWillShowNotification, object: nil)
+    NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillHide), name: UIResponder.keyboardWillHideNotification, object: nil)
+  }
+
+  @objc func dismissKeyboard() {
+    view.endEditing(true)
+    view.frame.origin.y = 0
```

Comment:
I think this line is related to the problem you told me about: views becoming black when keyboard is closing.
If this problem is still happening, we can't merge safely this branch, otherwise Anne will not be able to test properly the app..